Containers/recording/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.11.3-alpine3.18
+
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=664 recording.conf /etc/recording.conf
+
+RUN set -ex; \
+    apk add --no-cache \
+        ca-certificates \
+        tzdata \
+        bash \
+        ffmpeg \
+        libpulse \
+        bind-tools \
+        netcat-openbsd \
+        git \
+        wget \
+        shadow; \
+    # xvfb firefox chromium chromium-chromedriver?
+    # apk add --no-cache geckodriver --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing; \
+    useradd --system recordning; \
```

Comment:
```suggestion
    useradd --create-home --system recording; \
```

Reply:
why? I've copied this 1 to 1 from talk

Reply:
username is fixed

Reply:
I got this from here: https://github.com/nextcloud/spreed/blob/5656d1b52b756c7931a6f49518bc9de36ed9f816/recording/docs/installation.md?plain=1#L52